Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -688,15 +688,21 @@ public Declaration FindMemberEnclosingModule(Declaration callingModule, Declarat
 
         public Declaration FindMemberEnclosingProcedure(Declaration enclosingProcedure, string memberName, DeclarationType memberType, ParserRuleContext onSiteContext = null)
         {
```

Comment:
It looks like the onSiteContext parameter can be removed.

Reply:
Hmm. Indeed. Must have been an artifact left over from some past changes. The 4 references don't use this. Removed. 👍